src/docs/references/transaction-process-actions/index.md

```diff
@@ -581,7 +589,9 @@ Action to publish any reviews in the transaction.
 
 **Configuration options**: -
 
-### :action/reveal-customer-protected-data
+### Protected data
```

Comment:
I'd call this "Extended data". In my open PR there's already an action that deals with metadata. Maybe no need for separate heading for each type of extended data?